ui/src/timeMachine/components/TimeMachineFluxEditor.tsx

```diff
@@ -63,6 +78,56 @@ class TimeMachineFluxEditor extends PureComponent<Props> {
       </div>
     )
   }
+
+  private get rightDivision(): JSX.Element {
+    const {displayFluxFunctions} = this.state
+
+    if (displayFluxFunctions) {
```

Comment:
It looks like this and most of the JSX starting on line 106 is the same. Could you inline the shared parts into the render method on line 54, and leave the getter to only be for the parts that are different? (`FluxFunctionsToolbar` vs `VariablesToolbar`)